scripts/buildx_docker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
Should we keep both this script and `build_docker.sh`?

Reply:
I tried to DRY them out, but my thinking was both for now.